packages/react-checkbox/src/components/Checkbox/examples/Checkbox.Other.Example.tsx

```diff
@@ -1,7 +1,6 @@
 import * as React from 'react';
-import { Checkbox, ICheckboxProps } from '@fluentui/react-next/lib/Checkbox';
-import { Link } from '@fluentui/react-next/lib/Link';
-import { Stack } from '@fluentui/react-next/lib/Stack';
+import { Link, Stack } from 'office-ui-fabric-react';
```

Comment:
Maybe separate into `Link` and `Stack` specifically?

```suggestion
import { Link } from 'office-ui-fabric-react/lib/Link';
import { Stack } from 'office-ui-fabric-react/lib/Stack';
```